shared/src/iosMain/kotlin/com/projectnewm/example/di/ExampleNetworkModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.projectnewm.example.di
```

Comment:
I think this file can go in commonMain?

Reply:
Or is it not because it has an android version?  In which case should this be an `actual` and have an expect?

Reply:
Seems like we need to keep all the hilt stuff on Android side so we wouldn't have this in shared code for Android

Reply:
We can revisit if we need another DI solution that works for KMM